plugins/Delay/DelayControlsDialog.cpp

```diff
@@ -22,10 +22,13 @@
  *
  */
 
-#include "delaycontrolsdialog.h"
-#include "delaycontrols.h"
+#include "DelayControlsDialog.h"
+#include "DelayControls.h"
 #include "embed.h"
 #include "TempoSyncKnob.h"
+#include "../Eq/EqFader.h"
```

Comment:
Thinking about this more, moving this to the core code may be confusing. When the faders are rewritten for 2.0 my intention is to change the faders in my plugins to use the new ones, to keep consistency. So i guess the question is what is better, ````include "../Eq/EqFader.h"````, or copy the class to the delay folder breaking the dry principle but keeping the plugin non dependant on other plugins?